### ISSUE TITLE
Kotlin `ActivityStubExt.execute*` extension functions wrap arguments in list

### ISSUE THREAD

**Issue by rgrochowicz** [temporalio/sdk-java#2018]
## Expected Behavior

When I use the `ActivityStubExt.execute` extension function, I expect the `args` parameter to be given as-is to the underlying `ActivityStub.execute` method (keeping the varargs intact).

## Actual Behavior

The varargs `args` parameters to `ActivityStubExt.execute` is wrapped in a list rather than provided as a varargs parameter to `ActivityStub.execute`.

## Steps to Reproduce the Problem

  1. Create a new ActivityStub via `Workflow.newUntypedActivityStub`
  1. Import the Kotlin extension method via `import io.temporal.workflow.execute`
  1. Call `activityStub.execute<Void>("activityName", object { val foo = 1 })`
  1. Observe that the arguments were wrapped in a list rather than given directly to `ActivityStub.execute`

More complete example:

```kotlin
import io.temporal.workflow.execute
import io.temporal.activity.ActivityOptions

// ... within workflow ...

val activityStub = Workflow.newUntypedActivityStub(ActivityOptions {})
activityStub.execute("activityName", object { val foo = 1 })

```

The `ActivityTaskScheduled` event has an input of:

```json
[
  [
    {
      "foo": 1
    }
  ]
]
```

Rather than:

```json
[
  {
    "foo": 1
  }
]
```

The incorrect input is wrapped in an extra list.

To fix, I suspect that the call to `ActivityStub.execute` with varargs in [ActivityStubExt.kt](https://github.com/temporalio/sdk-java/blob/master/temporal-kotlin/src/main/kotlin/io/temporal/workflow/ActivityStubExt.kt#L36) needs to use the spread operator (`*`).

## Specifications

  - Version: Java SDK v1.23.0
  - Platform: macOS / Linux

**Comment by Quinn-With-Two-Ns** [comment]
Yes I think we are missing a spread operator, thank you for reporting this. I will follow up with my team and consider if there are any backwards compatibility issues with fixing this.

**Comment by Quinn-With-Two-Ns** [comment]
https://github.com/temporalio/sdk-java/pull/2395/files
